src/routes/GroupSettings/GroupSettingsTab/GroupSettingsTab.js

```diff
@@ -97,12 +99,12 @@ export default class GroupSettingsTab extends Component {
   save = async () => {
     this.setState({ changed: false })
     const { group, fetchLocation } = this.props
-    let coordinateLocationId
+    let locationId = this.state.edits.locationId
     if (group && this.state.edits.location !== group.location) {
-      coordinateLocationId = await ensureLocationIdIfCoordinate({ fetchLocation, location: this.state.edits.location, locationId: this.state.edits.locationId })
+      locationId = await ensureLocationIdIfCoordinate({ fetchLocation, location: this.state.edits.location, locationId })
```

Comment:
Do these changes impact the function of the code, or does it just rearrange variables in a cosmetic way?

Reply:
just cleanup as I was playing around with it